FastSimulation/Decayer/interface/Decayer.h

```diff
@@ -0,0 +1,36 @@
+#ifndef FASTSIM_DECAYER_H
+#define FASTSIM_DECAYER_H
+
+#include <memory>
+#include <vector>
+
+namespace gen {
+  class P8RndmEngine;
+}
+
+namespace CLHEP {
+  class HepRandomEngine;
+}
+
+namespace Pythia8 {
+  class Pythia;
+}
+
+namespace fastsim
+{
+    class Particle;
+    class Decayer 
+    {
+    public:
+	
+	Decayer();
+	~Decayer();
+	void decay(const Particle & particle,std::vector<std::unique_ptr<Particle> > & secondaries,CLHEP::HepRandomEngine & engine) const;
+	
+    private:
+	
+	std::unique_ptr<Pythia8::Pythia> pythia_; 
```

Comment:
It seems like Lukas did not want to explicity include <Pythia8/Pythia.h> and "Pythia8Plugins/HepMC2.h" in the Decayer.h header file. If you want me to use a simple member variable I would have to do that. What do you think about it?